android/modules/ui/src/java/ti/modules/titanium/ui/PickerRowProxy.java

```diff
@@ -29,6 +30,18 @@ public PickerRowProxy()
 	}
 
 	@Kroll.getProperty @Kroll.method
+	public String getColor()
+	{
+		return (String)getProperty(TiC.PROPERTY_COLOR);
```

Comment:
The formatting of casts is inconsistent throughout, so it's down to personal preference.